fuse/data/utils/split.py

```diff
@@ -138,10 +137,9 @@ def dataset_balanced_division_to_folds(dataset: DatasetBase, output_split_filena
             keys = [get_sample_id_key(), id]
         if keys_to_balance is not None:
             keys += keys_to_balance
-        df = ExportDataset.export_to_dataframe(dataset, keys, workers = workers, mp_context=mp_context)
-        df_folds = balanced_division(df, id, keys_to_balance, nfolds, **kwargs)
+        df = ExportDataset.export_to_dataframe(dataset, keys)
```

Comment:
revert

Reply:
what was the reason for these reverts?

Reply:
I meant that I should revert my modifications.
So you can ignore this one.